src/types/stardust/block/output/foundry.rs

```diff
@@ -21,6 +21,13 @@ impl FoundryId {
     const LENGTH: usize = iota::FoundryId::LENGTH;
 }
 
+impl FoundryId {
+    /// Get an implicit (zeroed) foundry ID, for new foundry outputs.
+    pub fn implicit() -> Self {
+        Self([0; Self::LENGTH])
+    }
+}
+
```

Comment:
I'm sorry for leading you down the wrong path, but foundries don't have an implicit ID. Only aliases and NFTs do. We therefore also need to find a different way to compute _created_ foundries.